src/data_manager/src/data_definition.rs

```diff
@@ -309,7 +309,7 @@ fn columns_table_types() -> [ColumnDefinition; 5] {
     ]
 }
 
-type InnerId = u64;
```

Comment:
I'd suggested to create separate type alias. I gave it `Inner` prefix to emphasise internal usage for `data_manager` only